### PR TITLE
Fixes Vitrium Frothe metabolization issues.

### LINKED ISSUE #21033: Vitrium Froth doesn't metabolize.
One of the new chemicals (Vitrium Froth) from the ash flora (cactus and mushroom leaves) doesn't really metabolize, it'll just stay in you forever, with the healing effects.

Also, are mushroom bowls suppose to do anything?

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
Fixes #21033

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 1]
```diff
@@ -475,10 +475,10 @@
 		. = 1
 	if(prob(20))
 		M.losebreath += 4
-	M.adjustBrainLoss(2*REM)
-	M.adjustToxLoss(3*REM,0)
-	M.adjustStaminaLoss(10*REM,0)
-	M.blur_eyes(5)
+		M.adjustBrainLoss(2*REM)
+		M.adjustToxLoss(3*REM,0)
```
Comment (phil235) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]: . = 1  because we're dealing damage

**Comment by Aloraydrel** [comment]
stupid bone man

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 2]
```diff
@@ -500,7 +501,8 @@
 	color = "#d3a308"
 	nutriment_factor = 3 * REAGENTS_METABOLISM
 
-datum/reagent/consumable/vitfro/on_mob_life(mob/living/M)
+/datum/reagent/consumable/vitfro/on_mob_life(mob/living/M)
 	if(prob(80))
 		M.adjustBruteLoss(-1*REM, 0)
 		M.adjustFireLoss(-1*REM, 0)
```
Comment (ChangelingRain) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]: . = 1 for the same reason

File: code/modules/reagents/chemistry/reagents/food_reagents.dm (hunk 1 above)
Comment (optimumtact) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]: . = TRUE

File: code/modules/reagents/chemistry/reagents/food_reagents.dm (hunk 2 above)
Comment (optimumtact) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]: . = TRUE

**Comment by TechnoAlchemisto** [comment]
Set it to true, also fixed an issue with bowl pathing that was reported in the same issue.

**Comment by TechnoAlchemisto** [comment]
uh oh

**Comment by optimumtact** [comment]
Thank u mr skeletal

**Pull request merged**